src/sentry/tasks/releasemonitor.py

```diff
@@ -260,6 +258,7 @@ def adopt_releases(org_id, totals):
                                     "sentry.tasks.process_projects_with_sessions.skipped_update"
                                 )
                                 capture_exception(exc)
+                        adopted_ids.append(rpe.id)
```

Comment:
If we fail to create the `rpe` in the second try/catch it will be unassigned here and fail. We should probably assign `rpe = None` and check that it has a value here before we append it.

We should probably have a test for this case.

Reply:
Writing a test 👍